docs/README.md

```diff
@@ -27,7 +27,7 @@ The site is viewable at `http://localhost:4000/derby/`.
 One-time container creation:
 
 ```
-docker run --name derby-docs-ruby -v "$(pwd)/docs:/derby-docs" -p 127.0.0.1:4000:4000 ruby:2.7 bash -c 'cd derby-docs && bundle install && bundle exec jekyll serve -H 0.0.0.0 -P 4000 --trace'
+docker run --name derby-docs-ruby -v "$(pwd)/docs:/docs" -p 127.0.0.1:4000:4000 ruby:2.7 bash -c 'cd docs && bundle install && bundle exec jekyll serve -H 0.0.0.0 -P 4000 --trace'
```

Comment:
Let's keep the container-internal path as `/derby-docs`, since there's no containing parent `derby` directory inside the container.